site/docs/encoding/axis.md

```diff
@@ -154,7 +154,7 @@ Axis configuration defines default settings for axes. Properties defined under t
 
 Additional property blocks can target more specific axis types based on the orientation (`"axisX"`, `"axisY"`, `"axisLeft"`, `"axisTop"`, etc.), band scale type (`"axisBand"`), or scale's data type (`"axisQuantitative"` and `"axisTemporal"`). For example, properties defined under the `"axisBand"` property will only apply to axes visualizing `"band"` scales. If multiple axis config blocks apply to a single axis, type-based options take precedence over orientation-based options, which in turn take precedence over general options.
 
-An axis configuration supports all [axis properties](#properties) except `position`, `orient`, `format`, `tickCount`, `values`, and `zindex`.
+An axis configuration supports all [axis properties](#properties) except `position`, `orient`, `format`, `values`, and `zindex`.
```

Comment:
also labelExpr?

Reply:
labelExpr is not an exception anymore.  (It was, but not listed, so nothing to remove.)